package.json

```
@@ -197,4 +188,4 @@
   ],
   "license": "SEE LICENSE IN ./LICENSE",
   "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)